Lib/test/test_difflib.py

```diff
@@ -466,13 +467,38 @@ def _assert_type_error(self, msg, generator, *args):
             list(generator(*args))
         self.assertEqual(msg, str(ctx.exception))
 
+class TestJunkAPIs(unittest.TestCase):
+    def test_is_line_junk_true(self):
+        for line in ['#', '  ', ' #', '# ', ' # ', '']:
+            is_junk = difflib.IS_LINE_JUNK(line)
+            self.assertTrue(is_junk)
```

Comment:
Why not just
```
self.assertTrue(difflib.IS_LINE_JUNK(line))
```
?

And you can add the second argument of `assertTrue()` for getting more information if the test fails:
```
self.assertTrue(difflib.IS_LINE_JUNK(line), rept(line))
```
But don't do this in test_is_line_junk_REDOS.